### ISSUE TITLE
Searchbox: form field element should have an id or name attribute

### ISSUE THREAD

**Issue by jnweiger** [opencloud-eu/web#2549]
### Initial Checklist

- [x] I understand this is a bug report and questions should be posted in the [Community Discussions](https://github.com/orgs/opencloud-eu/discussions/)
- [x] I searched [issues](https://github.com/opencloud-eu/web/issues?q=is%3Aissue) and couldn’t find anything (or linked relevant results below)

### Bug Description

oc-6.2.0 started at hetzner cloud via docker compose according to https://docs.opencloud.eu/docs/admin/getting-started/container/docker-compose/docker-compose-base/

Browser: Chromium 147.0.7727.137 openSUSE 15.6

This is a minor warning only, only seen in Chromium and only when clicking on the little red flag:

A form field element has neither an id nor a name attribute. This might prevent the browser from correctly autofilling the form.

To fix this issue, add a unique id or name attribute to a form field. This is not strictly needed, but still recommended even if you have an autocomplete attribute on the same element.

```
<input class="oc-search-input oc-input p-4 rounded-4xl disabled:cursor-not-allowed focus:bg-none focus:outline focus:outline-offset-2 focus:outline-white leading-7 h-8" aria-label="Suchbegriff eingeben" placeholder="Suchbegriff eingeben">

```

### Reproduction Steps

open web browser on main entry page.

### Expected Outcome

no browser warnings.

### Actual Outcome

Warnings, where the user cannot do anything about them.
